spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -0,0 +1,86 @@
+describe 'POST api/v1/targets', type: :request do
+  let(:user)            { create(:user) }
+  let(:topic)           { create(:topic) }
+  let(:target)          { Target.last }
+  let(:failed_response) { 400 }
+
+  describe 'POST create' do
+    subject { post api_v1_targets_path, params:, headers: auth_headers, as: :json }
+
+    let(:title)           { 'test' }
+    let(:radius)          { 27_384.4 }
+    let(:lat)             { -94.5566 }
+    let(:lng)             { -94.5566 }
+    let(:topic_id)        { topic.id }
+
+    let(:params) do
+      {
+        target: {
+          title:,
+          radius:,
+          lat:,
+          lng:,
+          topic_id:
+        }
+      }
+    end
+
+    it 'returns a successful response' do
+      subject
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'creates the target' do
+      expect { subject }.to change(Target, :count).by(1)
```

Comment:
To ensure only one is created you can do

```suggestion
      expect { subject }.to change(Target, :count).from(0).to(1)
```

Reply:
I agree that it is more exactly.